CHANGES.md

```diff
@@ -6,6 +6,55 @@ Version 0.3.0
 
 To be released.
 
+ -  `Bencodex.Types.Dictionary.SetItem()` became
+    to have more overloads.  [[#7]]
+     -  The type `Bencodex.Types.Dictionary.SetItem()` returns became
+        `Bencodex.Types.Dictionary`, not `IImmutableDictionary<IKey, IValue>`,
+        except for `Bencodex.Types.Dictionary.SetItem(IKey, IValue)`.
```

Comment:
How about this?

> Added overloads, which is listed below, return `Bencodex.Types.Dictionary` instead of `IImmutableDictionary<IKey, IValue>`.  Note that existing `SetItem(IKey, IValue)` method which implements `IImmutableDictionary<IKey, IValue>` is still remained as it had been.